scripts/ssl-poodle.nse

```diff
@@ -77,6 +77,8 @@ local function ctx_log(level, protocol, fmt, ...)
 end
 
 local function try_params(host, port, t)
+  t.host = host --adds host also so that host.registry is accessible inside tls.lua
```

Comment:
This isn't necessary, since `ssl-poodle` doesn't add discovered ciphers to the registry (and it probably shouldn't, because it doesn't check every cipher). Instead, we should be *retrieving* the stored results if they are there, probably in `find_ciphers`. That function would either perform its current function if there are no stored results, or it would filter the already-discovered ciphers down to CBC ciphers only and return those.

In order to do this, we'd have to store the discovered ciphers for each protocol separately (probably a good idea) something like this:

```lua
host.registry.tls_protos = {
  ["SSLv3"] = {
    "TLS_DHE_DSS_WITH_3DES_EDE_CBC_SHA",
    "TLS_RSA_WITH_3DES_EDE_CBC_SHA",
  },
  ["TLSv1.0"] = {
    "TLS_DHE_DSS_WITH_3DES_EDE_CBC_SHA"
  }
}
```